bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/wip/bpmn2java/ConverterXmlFileReaderTest.java

```diff
@@ -94,6 +94,22 @@ public void mapAndWriteTestNew() {
 //        );
     }
 
+    @Test
```

Comment:
BTW, what is cz.cvut.kbss.bpmn2stamp.**wip**.bpmn2java ? What is wip ? Work in progress ?

Reply:
removed in further commits